packages/terra-overlay/src/Overlay.jsx

```diff
@@ -208,7 +208,10 @@ class Overlay extends React.Component {
       <Portal>
         {backgroundScrollContent}
         <FocusTrap>
-          {overlayComponent}
+          {/* div addresses child focus change introduced in focus-trap-react v5.0.0 */}
+          <div>
```

Comment:
Do any of the styles applied to the overlay need to be applied here as well? https://github.com/cerner/terra-core/pull/2372/files#diff-5c1dfe9c477f29f0ae7f7738c5cb13e9R174

I'm thinking things like the fullscreen properties might be impacted by adding this wrapper.

Reply:
Turns out the `is-fullscreen` [selector](https://github.com/cerner/terra-core/blob/master/packages/terra-overlay/src/Overlay.module.scss#L34) that I thought might have issues....isn't actually used anywhere. 🤷‍♂